languages.toml

```diff
@@ -2208,6 +2208,19 @@ indent = { tab-width = 2, unit = "  " }
 name = "yuck"
 source = { git = "https://github.com/Philipp-M/tree-sitter-yuck", rev = "9e97da5773f82123a8c8cccf8f7e795d140ed7d1" }
 
+[[language]]
+name = "prql"
+scope = "source.yprql"
```

Comment:
Is this a typo or is the "y" intentional?

Reply:
Typo, will fix that later